tx-source/site_core.php

```diff
@@ -484,6 +484,7 @@
 $Definition['Connect your profile to social networks.'] = 'Connect your profile to social networks to share activity with your friends and followers.';
 $Definition['Contact Us'] = 'Contact Us';
 $Definition['Content'] = 'Content';
+$Definition['Content from Ignored User.'] = 'Content from Ignored User.';
```

Comment:
why is "from" small cased? should it be "From"?

Reply:
This is correct per Sarah's feedback, see the article https://prowritingaid.com/list-of-words-not-capitalized-in-titles . The rest is addressed.